src/app/design/guidelines/user-assistance/index.html

```diff
@@ -107,6 +107,14 @@ <h3 class="sky-subsection-heading">
 						</p>
 						</sky-repeater-item-content>
 				</sky-repeater-item>
+        <sky-repeater-item [isExpanded]="false">
+					<sky-repeater-item-title>Tile-level help</sky-repeater-item-title>
+					<sky-repeater-item-content>
+						<p>
+							For tasks where users require documentation for a tile, they invoke help from the tile header. If only a small number of items in the tile need explanation, use element-level help on those items instead. In general you should not combine tile-level help and element-level help on items within the tile.
```

Comment:
I'd make the same edits to the text here as on the tile component page. ... Also, should the first instance of "tile" be a link to the tile component docs? In the preceding section, we link to the modal component topic, so just checking.

Reply:
updated